tdrs-backend/tdpservice/parsers/validators/test/test_category3.py

```diff
@@ -500,54 +500,6 @@ def test_sumIsLarger():
     assert result.field_names == ['TestField1', 'TestField3']
 
 
-def test_validate__FAM_AFF__SSN():
```

Comment:
probably could have removed this in #3594 but just a test so no breaking changes.